docs/labs/installer.rst

```diff
@@ -57,207 +57,14 @@ You may also be interested in the long-form documentation if you like to see exa
-
-Deploy and Migrate a Stateful App
-=================================
-
-Now you will deploy a highly sophisticated stateful app to test out Flocker.
-
-We need to find out the IP addresses of our first two nodes.
-Do this by running:
-
-.. prompt:: bash $
-
-   cat cluster.yml
-
-Copy and paste the public IP addresses of the first two ``agent_nodes``.
-
-In this example, ``demo`` is the name of the Flocker volume being created, which will map onto the Flocker volume being created.
-
-.. prompt:: bash $
-
-    NODE1="<node 1 public IP>"
-    NODE2="<node 2 public IP>"
-    KEY="<path on your machine to your .pem file>"
-    chmod 0600 $KEY
-    ssh -i $KEY root@$NODE1 docker run -d -v demo:/data --volume-driver=flocker --name=redis redis:latest
-    ssh -i $KEY root@$NODE1 docker run -d -e USE_REDIS_HOST=redis --link redis:redis -p 80:80 --name=app binocarlos/moby-counter:latest
-    uft-flocker-volumes list
-
-This may take up to a minute since Flocker is provisioning and attaching an volume from the storage backend for the Flocker ``demo`` volume.
-At the end you should see the volume created and attached to the first node.
-
-Now visit ``http://<node 1 public IP>/`` and click around to add some Moby Docks to the screen.
-Now let's stop the containers, then start the stateful app on another node in the cluster.
-
-.. prompt:: bash $
-
-    ssh -i $KEY root@$NODE1 docker rm -f app
-    ssh -i $KEY root@$NODE1 docker rm -f redis
-    ssh -i $KEY root@$NODE2 docker run -d -v demo:/data --volume-driver=flocker --name=redis redis:latest
-    ssh -i $KEY root@$NODE2 docker run -d -e USE_REDIS_HOST=redis --link redis:redis -p 80:80 --name=app binocarlos/moby-counter:latest
-    uft-flocker-volumes list
-
-At the end you should see the volume has moved to the second node.
-
-This may take up to a minute since Flocker is ensuring the volume is on the second host before starting the container.
-
-Now visit ``http://<node 2 public IP>/`` and you’ll see that the location of the Moby Docks has been preserved.
-Nice.
-
-Cleaning up your Cluster
-========================
-
-When you're done, if you want to clean up, run the following steps to clean up your volumes, your instances and your local files:
-
-.. prompt:: bash $
-
-    ssh -i $KEY root@$NODE2 docker rm -f app
-    ssh -i $KEY root@$NODE2 docker rm -f redis
-    uft-flocker-volumes list
-    # Note the dataset id of the volume, then destroy it
-    uft-flocker-volumes destroy --dataset=$DATASET_ID
-    # Wait for the dataset to disappear from the list
-    uft-flocker-volumes list
-    # Once it's gone, go ahead and delete the nodes
-    uft-flocker-destroy-nodes
-    cd ~/clusters
-    rm -rf test
-
-.. note::
-
-    If you wish to clean up your cluster manually, be sure to delete the instances that were created in your AWS console and the ``flocker_rules`` security group.
+Get Started
+===========
+
+.. toctree::
+   :maxdepth: 1
+   
+   installer-getstarted
```

Comment:
the thinking was to keep the main page as the concept of the Installer, and its benefits, then the user links down to more information - to get started, then a tutorial, then clean up.

this will be tested before the installer gets moved into the main supported docs, so it might get concatenated again when that happens.